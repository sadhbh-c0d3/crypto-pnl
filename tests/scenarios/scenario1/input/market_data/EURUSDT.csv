1654077300000,1.15000000,1.20000000,1.15000000,1.20000000,,,,,,
1654077600000,1.20000000,1.20000000,1.20000000,1.20000000,,,,,,
1654077900000,1.25000000,1.25000000,1.20000000,1.25000000,,,,,,
1654078200000,1.25000000,1.25000000,1.20000000,1.20000000,,,,,,