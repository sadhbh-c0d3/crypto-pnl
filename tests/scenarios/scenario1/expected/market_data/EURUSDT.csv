1654077600000,1.20000000,1.20000000,1.20000000,1.20000000
